sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/NonFlagsEnumInBitwiseOperationTest.cs

```diff
@@ -30,21 +31,17 @@ public class NonFlagsEnumInBitwiseOperationTest
     {
         [TestMethod]
         [TestCategory("Rule")]
-        public void NonFlagsEnumInBitwiseOperation()
-        {
+        public void NonFlagsEnumInBitwiseOperation() =>
             Verifier.VerifyAnalyzer(@"TestCases\NonFlagsEnumInBitwiseOperation.cs",
-                new NonFlagsEnumInBitwiseOperation());
-        }
+                                    new NonFlagsEnumInBitwiseOperation(),
+                                    additionalReferences: MetadataReferenceFacade.GetSystemComponentModelPrimitives());
 
         [TestMethod]
         [TestCategory("CodeFix")]
-        public void NonFlagsEnumInBitwiseOperation_CodeFix()
-        {
-            Verifier.VerifyCodeFix(
-                @"TestCases\NonFlagsEnumInBitwiseOperation.cs",
-                @"TestCases\NonFlagsEnumInBitwiseOperation.Fixed.cs",
-                new NonFlagsEnumInBitwiseOperation(),
-                new NonFlagsEnumInBitwiseOperationCodeFixProvider());
-        }
+        public void NonFlagsEnumInBitwiseOperation_CodeFix() =>
```

Comment:
hm, how come this doesn't need the `GetSystemComponentModelPrimitives` references?

Reply:
I'm not sure. The first test is running the rule and fails because it cannot resolve `DesignerSerializationVisibility`. The second one is testing the code fix provider and the `Verifier.VerifyCodeFix` will not check for build errors as `Verifier.VerifyAnalyzer` does.